src/pages/stacks-blockchain/technical-specs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Technical Specifications
+description: Summary of technical specifications of Stacks 2.0
+---
+
+## Consensus
+
+- Proof of Transfer (PoX) as described in [SIP-007](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-007-stacking-consensus.md)
+- Network will transition to Proof of Burn (PoB) as described in [SIP-001](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md) after 10 years. More details [here](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md).
+
```

Comment:
maybe link out to https://github.com/blockstack/docs/blob/master/src/pages/stacks-blockchain/proof-of-transfer.md for further reading?